Create notebook and md file in mlops-stacks to enable inference table monitoring (#57)
databricks/mlops-stack: {{cookiecutter.project_name}}/monitoring/README.md

# Setup Data Monitoring for Inference Table

> **Databricks data monitoring is currently in private review. Please `sign up` for the private review and get approved 

Reviewer: ```suggestion
**Databricks Data Monitoring is currently in Private Preview. Please `sign up` for the preview and get approved 
```

before proceeding.**

> This directory contains the notebook that can run in databricks workspace
> to set up data monitoring for prod inference table.

Reviewer: ```suggestion
This directory contains the notebook that can run in a Databricks workspace
to set up Data Monitoring for a production inference table.
```

## Prerequisites

### Unity Catalog
The table to be monitored must exist in Unity Catalog.

> If Unity Catalog is enabled for your workspace but the table exists in metastore, please copy the table to Unity Catalog.

Reviewer: ```suggestion
If Unity Catalog is enabled for your workspace but the table exists in the Hive metastore, please copy the table to Unity Catalog.
```

For example, the following code shows how to import your data to a Unity Catalog managed table
 - load data from metastore example_project_predictions
 - write data to Unity Catalog my_unity_catalog.my_schema.my_inference_table
 - convert prediction column to double type
 - convert tpep_pickup_datetime to timestamp type

>  - convert tpep_dropoff_datetime to timestamp type

Reviewer: Are these instructions relevant for showing how to import data into UC? Seems orthogonal

Author: It describes the example, I wanted to indicate that changes can also be applied when the table is copied to UC. 
Let me remove them.

```
from pyspark.sql.functions import col, to_timestamp

df = spark.table("hive_metastore.default.example_project_predictions") \
        .withColumn("prediction",col("prediction").cast("double")) \
        .withColumn("tpep_pickup_datetime",to_timestamp("tpep_pickup_datetime")) \
        .withColumn("tpep_dropoff_datetime",to_timestamp("tpep_dropoff_datetime"))
df.write.saveAsTable(
    name = "my_unity_catalog.my_schema.my_inference_table"
)
```

### Inference Table

The inference table is required to contain the following columns
- A column that contains the predictions. For regression problems, the column must be a subclass of `NumericType`.
- A column that contains the time of inference. This column must be of type `TimestampType`.
- A column that contains the model version used for inference.

## Usage

### 1. Fill in data monitoring wheel URL
The `SetUpDataMonitoringForInferenceTable.py` notebook can be found in the monitoring directory. Please complete TODO 

> and fill in the wheel_URL. The URL can be found from data monitoring private review user guide.

Reviewer: ```suggestion
and fill in the wheel_URL. The URL can be found from the Data Monitoring Private Preview user guide.
```
